microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -176,9 +242,9 @@ def _get_node_nic_info(self, node: Node, nic_list: List[str]) -> None:
         assert_that(self.is_empty()).is_false()
 
     def _get_default_nic(self, node: Node, nic_list: List[str]) -> str:
```

Comment:
The nic is important in our test code. Can you move this class into a file called `nic.py` in lisa level? We may refine the code later, but it's a beginning to share more logic like get mac address and other information of a nic.

Reply:
I would be honored

Reply:
Wow that was a lot easier than I expected. It would make sense to me if the Nic eventually is just accessible through the node class. Will send up the refactor soon after some testing.